server/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -108,12 +113,17 @@
 
     private BorrowedItem<EventLoopGroup> eventLoopGroup;
 
+    // Due to how the current Itests are emulating multiple nodes within the same JVM,
+    // an active-sessions map is hoisted out of PostgresWireProtocol so that it is available per node for production setup as well as for testing.
+    private final Map<PostgresWireProtocol.KeyData, Session> activeSessions = new HashMap<>();
```

Comment:
Shouldn't this use a ConcurrentHashMap? There will be concurrent access from multiple handlers